### PR TITLE
New logo, new social card, and page adjustments

### PR DESCRIPTION
**Pull request by caneco**
### 1️⃣ Added the band new logo for the **Laravel Worldwide Meetup**

image

### 2️⃣ Added a new social card according the next event so when someone share the link that will be promoted 😁 

image

### CODE REVIEW

File: resources/views/components/event-listing.blade.php [hunk 1]
```diff
@@ -39,9 +36,7 @@ class="relative inline-flex items-center px-4 py-2 border border-transparent tex
         <ul>
             @if ($event->hasSpeaker(1))
                 <x-event-list-speaker :event="$event" number="1"/>
-
             @endif
-
```
Comment (driesvints) [inline resources/views/components/event-listing.blade.php:44]: Think it's best to revert much of the white lines you've removed in this PR. They're there to give the code some breathing room.

**Comment by driesvints** [comment]
I'll check this out this evening locally 👍

**Review by driesvints — DISMISSED** [review]

**Comment by driesvints** [comment]
@caneco can you revert the white line removals before we merge this?

**Comment by freekmurze** [comment]
Thanks for your work on this @caneco 

@driesvints I'll quickly re-add the breathing space 🙂

**Comment by freekmurze** [comment]
Deployed!

**Pull request merged**